diff --git a/node_modules/@graphcommerce/magento-graphql/mesh/magentoOrdersResolvers.ts b/node_modules/@graphcommerce/magento-graphql/mesh/magentoOrdersResolver.ts
similarity index 100%
rename from node_modules/@graphcommerce/magento-graphql/mesh/magentoOrdersResolvers.ts
rename to node_modules/@graphcommerce/magento-graphql/mesh/magentoOrdersResolver.ts
diff --git a/node_modules/@graphcommerce/magento-graphql/plugins/magentoOrders.ts b/node_modules/@graphcommerce/magento-graphql/plugins/magentoOrders.ts
index 2eaa85d..91167db 100644
--- a/node_modules/@graphcommerce/magento-graphql/plugins/magentoOrders.ts
+++ b/node_modules/@graphcommerce/magento-graphql/plugins/magentoOrders.ts
@@ -20,7 +20,7 @@ export const meshConfig: FunctionPlugin<typeof meshConfigBase> = (
       ...baseConfig,
       additionalResolvers: [
         ...(baseConfig.additionalResolvers ?? []),
-        '@graphcommerce/magento-graphql/mesh/magentoOrdersResolvers.ts',
+        '@graphcommerce/magento-graphql/mesh/magentoOrdersResolver.ts',
       ],
     },
     graphCommerceConfig,
